DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/sanity_utils.py

```diff
@@ -277,71 +68,3 @@ async def check_ntp_sync(dev, devices_dict):
         dev.applog.info(f"NTP not synced {rc} {out}")
         return False
     return True
-
-
-async def check_infra_to_infra_ping(dev, devices_dict):
-    """
-    - Check infra to infra pings over vlan100 - 10.1.4.3 on infra1 and 10.1.4.2 on infra2.
-    """
-    if dev.type is not DeviceType.INFRA_SWITCH:
-        return True
-
-    for infra in devices_dict[DeviceType.INFRA_SWITCH]:
-        # no need to ping self
-        if infra.host_name == dev.host_name:
-            continue
-
-        # do a poing to this device
-        # get the ip
-        out = await IpAddress.show(
-            input_data=[{infra.host_name: [{"dev": "vlan100", "cmd_options": "-j"}]}],
-            parse_output=True,
-        )
-        assert (
-            out[0][infra.host_name]["rc"] == 0
-        ), f"Failed to get ip address for vlan100 on {infra.host_name}"
-
-        addresses = out[0][infra.host_name]["parsed_output"]
-        infra_ip = None
-        for addr in addresses[0]["addr_info"]:
-            if addr["family"] == "inet" and addr["scope"] == "global":
-                infra_ip = addr["local"]
-
-        if infra_ip is None:
-            dev.applog.info(f"Could not get IP address for vlan100 to {infra.host_name}")
-
-        rc = await tb_ping_device(dev, f"{infra_ip}", dump=True)
-        if rc != 0:
-            dev.applog.info(f"Failed to reach {infra.host_name} {rc}")
-            return False
-
-    return True
-
-
-async def check_poe_devices(dev, devices_dict):
```

Comment:
Can we leave PoE? You can replace line 327 to 
```
rc, out = await dev.run_cmd("which poectl")
```

As I understand in the future we might have PoE in the upstream.

Reply:
accepted